torchaudio/functional.py

```diff
@@ -1160,6 +1161,38 @@ def riaa_biquad(
     return biquad(waveform, b0, b1, b2, a0, a1, a2)
 
 
+def contrast(
+        waveform: Tensor,
+        enhancement_amount: float = 75.
+) -> Tensor:
+    r"""Apply contrast effect.  Similar to SoX implementation.
+    Comparable with compression, this effect modifies an audio signal to make it sound louder
+
+    Args:
+        waveform (Tensor): audio waveform of dimension of `(..., time)`
+        enhancement_amount (float): controls the amount of the enhancement
+            Allowed range of values for enhancement_amount : 0-100
```

Comment:
nit: I would have been tempted to scale this parameter to be between 0 and 1, instead of 0 and 100. Since sox is using 0 and 100, we could just keep that. Thoughts?

Reply:
I think we should keep it 0-100 , so that those who are using existing sox effects will find it easy when migrating to  functional . Let me know if we want to scale 0 to 1 . I will make the change

Reply:
Sounds good :)